cfn-resources/encryption-at-rest/cmd/resource/resource.go

```diff
@@ -32,8 +32,12 @@ import (
 	"go.mongodb.org/atlas-sdk/v20231115002/admin"
 )
 
-var CreateAndUpdateRequiredFields = []string{constants.RoleID, constants.CustomMasterKey, constants.RoleID, constants.ProjectID}
-var ReadAndDeleteRequiredFields = []string{constants.ProjectID}
+var (
+	CustomerMasterKey             = "AwsKmsConfig.CustomerMasterKeyID"
```

Comment:
knit: can we change CustomerMasterKey var name to CustomerMasterKeyID to keep it the same?

Reply:
Added in d0fa8b7fb04cc39607e46c4968eb586bb6ffd806